reference/features/synonyms.md

```diff
@@ -1,69 +1,85 @@
+}
+```
 
-## The One-way association
+A search for "cv" would return any documents that contain the strings "Résumé", "resumé", or "resume", without being affected by case or accent marks.
 
-This makes it possible to determine that a word will be synonymous with another but not the other way around.
+## One-way Association
 
-example:
+Use this when you want one word to be synonymous with another, but not the other way around.
 
 ```
 phone => iphone
 ```
 
-By searching `phone` you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, documents containing `phone` will not be shown in your results.
+By searching `phone`, you will get all results containing `iphone` with the same relevance. However, if you search for `iphone`, documents containing `phone` will not be returned in your results.
 
 #### Example
 
-To create a one-way synonym list this is the JSON that should be [added to the settings](/reference/api/synonyms.md#update-synonyms).
+To create a one-way synonym list, this is the JSON syntax that should be [added to the settings](/reference/api/synonyms.md#update-synonyms).
 
 ```json
 {
   "phone": ["iphone"]
 }
 ```
 
-## The multi-way association
+## Mutual Association
 
 By associating one or more synonyms with each other, they will be considered the same in both directions.
 
-example:
-
 ```
 shoe <=> boot <=> slipper <=> sneakers
 ```
 
-When a search is done with one of these words, all the others will be considered as the same word and will appear in the search results.
+When a search is done with one of these words, all synonyms will be considered as the same word and will appear in the search results.
 
-However, in the case of word to sentence or sentence to sentence
+#### Example
 
-example:
+To create a mutual association between four words, this is the JSON syntax that should be [added to the settings](/reference/api/synonyms.md#update-synonyms).
 
+```json
+{
+  "shoe": ["boot", "slipper", "sneakers"],
+  "boot": ["shoe", "slipper", "sneakers"],
+  "slipper": ["shoe", "boot", "sneakers"],
+  "sneakers": ["shoe", "boot", "slipper"]
+}
 ```
-"San Fransisco" <=> SF
-```
 
-The "San Fransisco" search will be considered less relevant than the "SF" search but will still be considered an acceptable search result in the absence of a more relevant result.
+## Multi-Word Phrases
+
+Take note that **multi-word phrases are treated differently** than associations between individual words.
+
+When a multi-word phrase is considered the synonym of another word or phrase, the **exact search query will always take precedence over its synonym(s)**.
 
 #### Example
 
-To create a multi-way synonym list this is the JSON that should be [added to the settings](/reference/api/synonyms.md#update-synonyms).
+Suppose you set "San Francisco" and "SF" as synonyms with a [mutual association](#mutual-association)
 
 ```json
 {
   "san francisco": ["sf"],
   "sf": ["san francisco"]
 }
 ```
+
```

Comment:
Could we add a little warning or a known limitation. Multi-Word Phrase is limited to three words. That's why `"The Lord of the Rings" => "TLOTR"` doesn't work.